galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -353,3 +353,12 @@ messages:
         type: string
       - name: dupno
         type: string
+
+  - name: "VirtualServiceHostNotFoundInGateway"
+    code: IST0132
+    level: Error
+    description: "Host defined in VirtualService not found in Gateway object."
+    template: "Host defined in VirtualService not defined in Gateway %s."
```

Comment:
The analyzer message should provide as much info as possible to rectify the problem. How about also add `host`, `virtual service`, and `gateway` name in the message?
 
Host  `<host name>` defined in VirtualService`<virtualservice name>` not found in Gateway `<gateway name>`.

Reply:
let me take a look for this

Reply:
this is done
```
Error [IST0132] (VirtualService service.default example.yaml:23) one or more host [bar.com haha.lol] defined in VirtualService default/service not defined in Gateway istio-system/gw.
```

Reply:
small correction https://github.com/istio/istio/pull/28486#discussion_r516590467